### DISCUSSION TITLE
Can't only create workflow/actions from web page?

### DISCUSSION THREAD

**Discussion by zillionare** [community/community#26607] [Actions]
<p>I’m creating a cookiecutter template for pypakcage, and I’d like to setup CI using github actions.<br>
if using travis, all you need is to put a traivis.yml into the repo, then push the file to github repo, CI is triggered. However, when I doing similar things with github actions, the workflow is not created.</p>
<p>by doing similiar things I mean, I have created .github folder at local, and put a workflow script there, then push it to gihthub. Then I checked from github web page, the script(main.yml) is actually there, but the actions are not triggered.</p>
<p>So the only way to enable github actions is to create it from web page?</p>
<p>If you would like to help, these are extra info may help:</p>
<ol>
<li>the cookiecutter template repo here <a href="https://github.com/zillionare/cookiecutter-pypackage" rel="noopener nofollow ugc">https://github.com/zillionare/cookiecutter-pypackage</a>
</li>
<li>the repo generated by cookiecutter: <a href="https://github.com/zillionare/test_0717_0318" rel="noopener nofollow ugc">https://github.com/zillionare/test_0717_0318</a>
</li>
<li>
</li>
</ol>

**Comment by airtower-luna** [comment, marked as answer]
<p>The workflow file in your repository has branch filters so it’ll only run for pushes or pull requests to <code>master</code>, but your default branch is called <code>main</code>. You’ll have to adjust the branch filters to when you actually want it to run.</p>

**Comment by zillionare** [comment]
<p>Works like a charm. after I change branch filte to [master, main], it worked.</p>
<p>Thanks!</p>
